client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/toggle-activate-monitoring/index.tsx

```diff
@@ -141,8 +141,11 @@ export default function ToggleActivateMonitoring( {
 		<>
 			<span
 				ref={ statusContentRef }
+				role="button"
```

Comment:
Added role and tabIndex because [Static HTML elements with event handlers require a role](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/93f78856655696a55309440593e0948c6fb96134/docs/rules/no-static-element-interactions.md)